keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentArtifactPin.kt

```diff
@@ -3,7 +3,6 @@ package com.netflix.spinnaker.keel.core.api
 import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
 
 data class EnvironmentArtifactPin(
-  val deliveryConfigName: String,
```

Comment:
This was the only way to look up the delivery config for authorization in the `@PreAuthorize` annotation:
https://github.com/spinnaker/keel/blob/ebad5af538bca45ec1d6a6c3a9c80946ec5ec53a/keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ArtifactController.kt#L78-L81